packages/cascara/src/ui/Form/__globals.js

```diff
@@ -0,0 +1,5 @@
+const WARNING_STRINGS = {
+  MISSING_STATE_FORM: 'Form requires dataDisplay to present a form component.',
```

Comment:
```suggestion
  INVALID_EDITING_AND_DISPLAY: 'Form requires a dataDisplay prop in order to also show isInitialEditing',
```

Since this message is for developers, it is okay for us to reference specific props and talk about how they should be used in combination